ITEM.tsv

```diff
@@ -3274,13 +3274,13 @@ ITEM_20150317_003273	Silver Dye for Magical Tattoo
 ITEM_20150317_003274	Silver dye used for Simon Shaw's tattoo. It is relatively easy to get.
 ITEM_20150317_003275	Cordelier's Parchment
 ITEM_20150317_003276	Parchment Cordelier gave you. Although coal powder has been used, it is hard to see the letters because the parchment has darkened.
-ITEM_20150317_003277	
-ITEM_20150317_003278	
+ITEM_20150317_003277	Nucleus of the platform Nuremberg Rem
+ITEM_20150317_003278	Power source to move the Porurunremu. It is necessary for the other case can not write brooch.
```

Comment:
Contains the power source that moves the Fallenlem. Grita, who can't use magic, needs it.